clients/drcachesim/tools/invariant_checker.h

```diff
@@ -91,12 +91,15 @@ class invariant_checker_t : public analysis_tool_t {
         bool found_instr_count_marker_ = false;
         uint64_t last_instr_count_marker_ = 0;
         std::string error;
+        // Track the location of errors.
+        memref_tid_t tid = -1;
+        uint64_t ref_count = 0;
```

Comment:
This seems more like a `ref_id`/`ref_ordinal` than `ref_count`

Reply:
I don't follow: it is not an "id".  It is the count of memref trace entries to this point in this thread.

Reply:
Right, so `ref_ordinal` makes the most sense then. IMHO, "ordinal" is clearer than "count", as the latter may be confused to mean the total count or something.

Reply:
But this is the per-thread counter that is used to count the references: this is not a stored snapshot for one error.  So this request, to me, implies that every counter variable would have to be named "ordinal" as it's not the total count until the very end when all the increments are done.

Reply:
Probably okay to keep it `ref_count`.